packages/slidev/node/export.ts

```diff
@@ -223,9 +224,18 @@ export async function exportSlides({
       preferCSSPageSize: true,
     })
 
+    // Edit generated PDF: add metadata and (optionally) TOC
+    let pdfData = fs.readFileSync(output)
```

Comment:
Can we use the promises version of `fs`? Thanks

Reply:
Sure! Done